src/server.js

```diff
@@ -26,7 +24,7 @@ module.exports = function (mozaik) {
 
     app.get('/', function (req, res) {
         res.render('index', {
-            env:           config.env,
+            env:           mozaik.config.env,
```

Comment:
I think it should be moved to another PR as it's not related to custom express instance